cli/pcluster/config/mappings.py

```diff
@@ -1030,6 +1031,7 @@
     "key": "cluster",
     "default_label": "default",
     "cluster_model": "HIT",
+    "validators": [shared_dir_and_ebs_settings_validator],
```

Comment:
Instead of adding this to two lists, we could make a list called CLUSTER_COMMON_VALIDATORS, add this to it, and specify it for both cluster model definitions.

Then in CLUSTER_SIT, the validators attribute could be set to `CLUSTER_COMMON_VALIDATORS + [cluster_validator]`, and for CLUSTER_HIT it woulds be just the common validators list.

Reply:
Good idea, done!